src/pushsource/_impl/model/vms.py

```diff
@@ -74,6 +74,16 @@ class VMIPushItem(PushItem):
     boot_mode = attr.ib(type=BootMode, default=None, validator=optional(in_(BootMode)))
     """Boot mode supported by the image (if known): uefi, legacy, or hybrid (uefi + legacy)."""
 
+    marketplace_title_template = attr.ib(type=str, default=None, validator=optional_str)
+    """The template is of the form used by ``str.format``, with available keywords being all of
```

Comment:
The doc string should still say something about what the attribute actually means, not just what the format is.

Given that there is a `marketplace_title` property, this attribute's doc string could mention that it's used to generate that value, and a sentence explaining what `marketplace_title` actually means could be in that property's doc string.

Reply:
I've changed it, PTAL again